src/applications/simple-forms/21-0972/pages/veteranIdentificationInformation2.js

```diff
@@ -1,9 +1,7 @@
 import {
   serviceNumberUI,
-  serviceNumberSchema,
   ssnSchema,
   ssnUI,
-  vaFileNumberSchema,
```

Comment:
i also removed this from the web component https://github.com/department-of-veterans-affairs/vets-website/pull/24710/files#:~:text=const%20SSN_DEFAULT_TITLE%20%3D%20%27Social%20Security%20number%27%3B, in this review via Pat's feedback that it shouldnt be part of the pattern, so seems were aligned here